code/frameworks/nextjs/template/cli/js/Header.jsx

```diff
@@ -46,7 +46,9 @@ export const Header = ({ user, onLogin, onLogout, onCreateAccount }) => (
 );
 
 Header.propTypes = {
-  user: PropTypes.shape({}),
+  user: PropTypes.shape({
```

Comment:
Should we be getting these from `renderers/react`?

Reply:
Was thinking about that, not sure, I just wanted to keep things as straight forward as possible in this PR